packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionScreen.tsx

```diff
@@ -40,7 +40,7 @@ const JurisdictionScreen = ({ setIsCheckBoxChecked }: TJurisdictionScreenProps)
             className={twMerge(
                 `flex flex-col w-full lg:p-16 items-center justify-between transition-all ease-in duration-[0.6s] p-40`,
                 isDynamicLeverageVisible &&
-                    '[transform:rotateY(-180deg)] h-[700px] opacity-0 bg-system-light-primary-background'
+                    '[transform:rotateY(-180deg)] opacity-0 bg-system-light-primary-background d-none lg:block'
```

Comment:
@thisyahlen-deriv noticed you created `d-none` style in tailwind config, but as I know you can use `hidden` to set `display: none`. It's not an error, but just let you know it's an extra wrapper around the original tailwind (https://tailwindcss.com/docs/display#hidden). You can ignore this comment, I already approved this PR

Reply:
I tried this @sergei-deriv 🙏 
![Screenshot 2024-03-05 at 5 18 26 PM](https://github.com/binary-com/deriv-app/assets/104053934/543f95c9-7abd-42c5-b609-340841eb3819)

But due to our core base styles, hidden property was not working 😞

Reply:
@thisyahlen-deriv thanks for the explanation!